test/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "target": "es5",
+    "module":"commonjs",
+    "lib": ["es2015", "es2016", "es2017"],
+    "strict": true,
+    "sourceMap": true,
+    "declaration": true,
+    "allowSyntheticDefaultImports": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "declarationDir": "dist",
+    "noImplicitAny": false,
+    "outDir": "dist",
+	"noEmit": true,
+    "typeRoots": [
+      "node_modules/@types"
```

Comment:
If I understand it correctly, this file was added to include tsx files. For my setup I needed to change the type root "node_modules/@types" to "../node_modules/@types". Else my IDE (IntelliJ) did not correctly recognize the jest types. Did it work for your setup ? I just want to make sure it is correct and not only a specific problem on my machine. I also think we should keep everything isolated and as small as possible. Therefore I like the idea of having a seperate tsconfig for tsx inclusion. What do you think about moving the ts/tsx inclusion test into test/integration/tsx and putting the tsconfig into tsx so it does only affect its scope ? Thanks for contributing! I opened up the issue section were discussions about enhancements can be held.

Reply:
Thank you for reviewing. I actually had to fix this to run the tests in intellij, however, on the command line it was working correctly. I can actually imagine more integration tests using tsx, so probably tsx-support is something which should in general be supported in integration tests, what do you think?